langwatch/src/pages/api/utils/index.test.ts

```diff
@@ -0,0 +1,357 @@
+import { describe, expect, it } from "vitest";
+
+import { getLatestUpdatedAt } from "./index";
+
+describe("utils test", () => {
+  it("gets the latest updated at time", async () => {
+    const latestUpdatedAt = getLatestUpdatedAt(traces);
```

Comment:
since we don't have an integration test for this feature, after this goes live, please set up some alerts for ourselves that will trigger every now and then, so we know if all good with the trigger system itself of it something annoys us